lib/device/library/linux.rb

```diff
@@ -2,22 +2,76 @@
 #
 # Copyright 2013 whiteleaf. All rights reserved.
 #
+# rubocop:disable Lint/AssignmentInCondition
+# rubocop:disable UtilityFunction
```

Comment:
__[rubocop]__
* Lint/UnneededDisable: Unnecessary disabling of `UtilityFunction` (unknown cop).

<!-- Comment by [SideCI](https://sideci.com) -->